android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/TiListView.java

```diff
@@ -55,7 +58,6 @@
 import android.widget.ListView;
```

Comment:
I have left an unused _FrameLayout_ import from https://github.com/appcelerator/titanium_mobile/pull/9353 before the _ListView_ import. Could you, please, remove it for me?